diff --git a/frameworks/cocos2d-x/cocos/scripting/js-bindings/manual/ui/jsb_cocos2dx_ui_manual.cpp b/frameworks/cocos2d-x/cocos/scripting/js-bindings/manual/ui/jsb_cocos2dx_ui_manual.cpp
index 07b338d..70f1898 100755
--- a/frameworks/cocos2d-x/cocos/scripting/js-bindings/manual/ui/jsb_cocos2dx_ui_manual.cpp
+++ b/frameworks/cocos2d-x/cocos/scripting/js-bindings/manual/ui/jsb_cocos2dx_ui_manual.cpp
@@ -701,9 +701,9 @@ void register_all_cocos2dx_ui_manual(JSContext* cx, JS::HandleObject global)
 
     JS_DefineFunction(cx, JS::RootedObject(cx, jsb_cocos2d_ui_PageView_prototype), "addEventListener", js_cocos2dx_UIPageView_addEventListener, 2, JSPROP_ENUMERATE | JSPROP_PERMANENT);
     
-    JS_DefineFunction(cx, JS::RootedObject(cx, jsb_cocos2d_ui_ScrollView_prototype), "addEventListener", js_cocos2dx_UIScrollView_addEventListener, 2, JSPROP_ENUMERATE | JSPROP_PERMANENT);
+    JS_DefineFunction(cx, JS::RootedObject(cx, jsb_cocos2d_ui_ScrollView_prototype), "addEventListenerScrollView", js_cocos2dx_UIScrollView_addEventListener, 2, JSPROP_ENUMERATE | JSPROP_PERMANENT);
 
-    JS_DefineFunction(cx, JS::RootedObject(cx, jsb_cocos2d_ui_ListView_prototype), "addEventListener", js_cocos2dx_UIListView_addEventListener, 2, JSPROP_ENUMERATE | JSPROP_PERMANENT);
+    JS_DefineFunction(cx, JS::RootedObject(cx, jsb_cocos2d_ui_ListView_prototype), "addEventListenerListView", js_cocos2dx_UIListView_addEventListener, 2, JSPROP_ENUMERATE | JSPROP_PERMANENT);
 
     JS_DefineFunction(cx, JS::RootedObject(cx, jsb_cocos2d_ui_LayoutParameter_prototype), "setMargin", js_cocos2dx_LayoutParameter_setMargin, 1, JSPROP_ENUMERATE | JSPROP_PERMANENT);
 
